lib/Synchronise/Document.hs

```diff
@@ -41,13 +43,14 @@ data Document = Document
 
 makeLenses ''Document
 
-instance ToJSON Document where
-    toJSON = _documentContent
+$(deriveJSON defaultOptions ''Document)
```

Comment:
The *Document functions need JSON instances for PostgreSQL. I dunno, what's the semantics of those functions?